src/redis_cache.rs

```diff
@@ -65,9 +74,13 @@ lazy_static! {
 use self::r2d2_redis::RedisConnectionManager;
 use self::r2d2_redis::{r2d2, redis};
 
+static CACHE_PREFIX: &str = "v2:cache";
+static TAG_PREFIX: &str = "v2:tag";
```

Comment:
Could we use a namespace for "v2" rather than hard coding?

Reply:
I'm not sure I understand what you mean. Can you give me an example?

Reply:
Since "v2" is specific to our environment we could use the configurable namespace field to version the keys instead of hardcoding. Just a thought :)

Reply:
Oh yes, good idea!